### DISCUSSION TITLE
Bug Notification filter

### DISCUSSION THREAD

**Discussion by 4Source** [community/community#126297] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Found not really a bug, but it is kind of bad information. If the filter starts with an ``space``  it says filter don't supported yet. Maybe add a trim in front so it could Say something like "did you mean unread?", "no leading space allowed" or just allow it.

This is not only for unread.

![image](https://github.com/community/community/assets/38220764/7544ba5a-3a41-4088-a70c-2a38df5a8072)

**Comment by nangerno** [comment]
It is not a bug, it means that it can't allow to show unread information to you yet.
Of course, you are right, But this function is in the progress of development now.

**Reply by 4Source** [reply]
> Found not really a bug, but it is kind of bad information.

Have you read. And not sure your understand or I doesn't know but it is not about the ``unread`` this is just a example this seems to happen for every filter(repo, is, reason, author, org) when after the ``:`` follows a " " space the message ``Sorry, we don't support the ... filter yet. Learn more about filters.`` is displayed I think it would be relative simple to check for the leading space and display different Message as mentioned.
